Sources/MongoMobile/MongoMobile.swift

```diff
@@ -103,19 +103,13 @@ public class MongoMobile {
      * Perform required operations to clean up the embedded server.
      */
     public static func close() throws {
-        self.embeddedClients.forEach { ref in
-            ref.reference?.close()
-        }
+        self.embeddedClients.forEach { ref in ref.reference?.close() }
+        embeddedClients.removeAll()
```

Comment:
In the Swift driver, we've been making explicit use of `self`. I'm not sure if that applies to this repository as well, though.